CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # x.x.x (next release)
 
+# 0.6.1
```

Comment:
We're actually working on `1.0.0`! You can drop the heading and leave this under the `x.x.x` section so we don't forget it.

We'll do the version bump after some more refactoring (specifically the cmd stuff - e.g. I think we'll replace `polaris --dashboard --dashboard-port=3000` with `polaris dashboard --port=3000`). I'll have to spend some time documenting all the changes and the upgrade path.

Reply:
Oh, to avoid duplicate work take a look at this, https://github.com/FairwindsOps/polaris/blob/bb/cobra-cmd/docs/usage.md